examples/kokkos/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+#------------------------------------------------------------------------------#
```

Comment:
Do we need this?

Reply:
All `CMakeLists.txt` files in the examples folder have the same header, either change them all (in a different PR) or leave it like this